File: app/src/main/java/netpro/keyTransmitter/EmptyKey.java
package netpro.keytransmitter;

public class EmptyKey extends Key {

    private static final long serialVersionUID = -1696906106555982373L;

    public EmptyKey() {
        super(1, 1, "", Type.EMPTY);
    }

    public EmptyKey(int columnSpan, int rowSpan, String description, Type type) {
        super(columnSpan, rowSpan, description, type);
    }

}

File: app/src/main/java/netpro/keytransmitter/EmptyKey.kt
package netpro.keytransmitter

/**
 * 空を表すボタンです
 */
class EmptyKey : Key {

    constructor() : super(1, 1, "", Key.Type.EMPTY)

    constructor(columnSpan: Int, rowSpan: Int, description: String, type: Key.Type) : super(columnSpan, rowSpan, description, type)

    companion object {
        private val serialVersionUID = -1696906106555982373L
    }

}
